### ISSUE TITLE
Dead links on https://dev-sec.io/project/

### ISSUE THREAD

**Issue by joubbi** [dev-sec/dev-sec.github.io#54]
**Describe the bug**
The page https://dev-sec.io/project/ contains links to best-known guides which gives a 404.

**Expected behavior**
I expected to read about security guides used by this project.

**Actual behavior**

```
The requested page could not be found.

Please try the following options:

    Check the address.
```

I figured that this is the best place to report this, since I couldn't find any other site admin contact information on the page.

**Comment by rndmh3ro** [comment]
Hi @joubbi,

I moved the issue to the correct repo.
The links should be updated.

**Comment by joubbi** [comment]
Thanks for fixing the links, but it still looks a little bad since the link to NSA is for RHEL 5.
This makes the whole project look outdated.

On December 19, 2020 7:16:50 AM UTC, Sebastian Gumprich <notifications@github.com> wrote:
>Closed #54.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/dev-sec/dev-sec.github.io/issues/54#event-4132223916

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
